src/tiff_decoder.ts

```diff
@@ -289,19 +289,22 @@ export default class TIFFDecoder extends IOBuffer {
     // General Image Dimensions
     const width = ifd.width;
     const height = ifd.height;
+    const rowsPerStrip = ifd.rowsPerStrip;
```

Comment:
No need to create a variable, can be inlined now

Reply:
Is it computed in a loop / many times? I see it used in a ternary when reading metadata.

It was just a nit to improve code readability, I'm fine with your implementation even if it does not affect performance.